test/e2e/e2e_test.go

```diff
@@ -427,9 +428,6 @@ func waitForImportToSucceed(
 	importStatusList, err := client.GetImportStatusesByTranslation(ts.ID)
 	require.Equal(t, 1, len(importStatusList))
 	importStatus := importStatusList[0]
-	if importStatus.StartAt == 0 {
```

Comment:
This is already checked below in "step 1", unsure why this is here.

Reply:
We don't need this.